### DISCUSSION TITLE
Please make a roadmap for this whole project.  Then the new developer can collaborate.

### DISCUSSION THREAD

**Discussion by ryihan** [dogecoin/dogecoin#2389] [General]

**Comment by AbcSxyZ** [comment]
To get a kind of "roadmap", you should take a look at the [projects section](https://github.com/dogecoin/dogecoin/projects) of the repo. You will get features in development for next releases.

From my pov, information are unclear, you're probably asking this question for this kind of reason. To let people collaborate, I think by having a better documentation, the project will be more able to let people know what's going on, and give them resources to facilitate their inclusion.
See [Docs4Doge](https://github.com/dogecoin/dogecoin/issues/2271).
